manifests/init.pp

```diff
@@ -26,25 +26,31 @@
   $anon_upload_enable      = 'NO',
   $anon_mkdir_write_enable = 'NO',
   $dirmessage_enable       = 'YES',
+  $dirlist_enable          = 'YES',
   $xferlog_enable          = 'YES',
   $connect_from_port_20    = 'YES',
+  $connect_timeout         = '60',
   $chown_uploads           = 'NO',
   $chown_username          = undef,
   $xferlog_file            = '/var/log/vsftpd.log',
   $xferlog_std_format      = 'YES',
   $idle_session_timeout    = '600',
   $data_connection_timeout = '120',
-  $nopriv_user             = undef,
+  $nopriv_user             = 'ftpsecure',
```

Comment:
for values that already exist, we should just set those in Hiera. We do want to submit this PR back to upstream.